### ISSUE TITLE
[Request] Laverna

### ISSUE THREAD

**Issue by linusg** [keeferrourke/la-capitaine-icon-theme#323]
### Application information

| Application name | Icon name  | Context     | Icon reference |
| ---------------- | :--------- | :---------- | :------------- |
| Laverna    | `laverna` | Markdown editor for notes | ![grafik](https://user-images.githubusercontent.com/19366641/35487471-39551060-047c-11e8-9d94-aa8647879920.png) |

### Desktop file

```
[Desktop Entry]
Name=Laverna
Exec=/opt/laverna/laverna
Icon=laverna  # until now: /opt/laverna/resources/app/dist/images/icon/icon-512x512.png
Terminal=false
Type=Application
StartupNotify=false
Categories=Office;
```

Hi Keefer,

I know you have limited time. But I love this icon theme so much, as you probably know, and I hate seeing single unthemed icons in my plank dock.

[Laverna](https://laverna.cc) isn't that popular (even though they have 7.3k stars on [their GitHub](https://github.com/Laverna/laverna)) and only has a packaged version for Arch. For all other distributions they provide binaries and installations from source, but there's not even a `.desktop` file included. I made my own, included above. So, the icon name can be chosen as there's no official one.

You'll get it, it's very unlikely that many people will need this Icon. It should be very simple to make (green square or circle, maybe a gradient and a checkmark), and I would really appreciate having one. But I would also understand if this is not a priority for you.

You end up with the decision, thanks for your terrific work in either case!

– Linus

**Comment by keeferrourke** [comment]
@linusg I just pushed a modified version of the gnome-todo icon with a light-green gradient. Let me know what you think :)

Cheers, and thanks for understanding my hectic schedule :P

**Comment by linusg** [comment]
TL; DR - Love it!

I just noticed I had a slightly outdated clone of the repo, after a `git pull` some other icons changed as well. 😄
I didn't even noticed Signal had no themed icon before... and Spotify is new. Looks more macOS-ish I guess.
So, this was fast, and for you maybe just a matter of changing the colors. Thanks again!

**Comment by linusg** [comment]
Just out of interest: is there any rule when you rotate the squared icons by a few degrees (e.g. Gimp, Inkscape) and when not? Or is this just another result of not strictly sticking to design guidelines as in https://github.com/keeferrourke/la-capitaine-icon-theme/issues/316? 😄

**Comment by Riotism** [comment]
@linusg We've just added [a section](https://github.com/keeferrourke/la-capitaine-icon-theme/wiki/Design-Guidelines#tilted-square) on that in our new design guidelines. Please tell us if we should list out more specifications 😃

**Comment by linusg** [comment]
Those guidelines are a great step forwards IMO. Still missing the "when or when not to tilt" though 😉
After all I sadly still lack advanced Inkscape knowledge, otherwise I would start making an icon or two right now 😐

**Comment by keeferrourke** [comment]
I think @Riotism has updated the design document to clarify :) While it was undocumented before, tilted square icons are meant for "creative" applications, they are just a bit more fun than conventional square ones.
